ispcrt/CMakeLists.txt

```diff
@@ -52,6 +52,8 @@ if (ISPCRT_BUILD_GPU)
   endif()
 endif()
 
+option(ISPCRT_BUILD_STATIC "Enable static library building" ON)
```

Comment:
Let's add `ispcrt` to the option description so it'll be aligned with others. 
`Enable static library building` -> `Build ispcrt static library`

Reply:
OK, reworded